config/jobs/kubernetes/sig-testing/kubernetes-kind.yaml

```diff
@@ -619,6 +619,8 @@ periodics:
         # GCE VM is used for setting up cgroup v2 and systemd.
         # (k8s-infra-prow-build lacks cgroup v2, and the kubekins-e2e container lacks systemd)
         (cd ; GO111MODULE=on go install github.com/rootless-containers/kubetest2-kindinv@master)
+        # Initialize the credentials for Google Cloud
+        gcloud auth activate-service-account --key-file="${GOOGLE_APPLICATION_CREDENTIALS}"
```

Comment:
example on how @justinsb set up the tests in cloud-provider gcp https://github.com/kubernetes/cloud-provider-gcp/blob/992c1c33f1be3f5204934e02b8488d117b50dd1c/test/boskos.sh#L54 

https://github.com/kubernetes/cloud-provider-gcp/blob/992c1c33f1be3f5204934e02b8488d117b50dd1c/e2e/scenarios/kops-simple#L32

Reply:
sorry, it was not meant for you the comment Akihiro

I´m not familiar neither with the best practices on these kind of jobs, so instead of keep fixing forward as we are doing now, I prefer to get some advices to avoid reaching a dead end

Reply:
@BenTheElder Could you take a look?

Reply:
We should be using [boskos](https://github.com/kubernetes-sigs/boskos) to rent project, because otherwise we risk leaking created VMs etc.

I'd expect that part to happen in the kubetest2 deployer though (see e.g. the GCE deployer, there's a Go Boskos client), and you'd still need the application credential active.

runner.sh should already handle activating the credential though https://github.com/kubernetes/test-infra/blob/49228c2a7e4a9bf613e826dc9fac5bf9c61195f7/images/bootstrap/runner.sh#L105

Reply:
Sure:
- https://github.com/kubernetes/test-infra/pull/31577